ddprof-ffi/src/vec.rs

```diff
@@ -154,7 +154,7 @@ mod test {
         assert!(ffi_vec.capacity >= 2);
 
         let slice = unsafe { ffi_vec.as_slice().as_slice() };
-        let first = slice.get(0).unwrap();
+        let first = slice.first().unwrap();
```

Comment:
I disagree with the lint in this specific case.

```rust
let first = slice.first().unwrap();
let second = slice.get(1).unwrap();
```

breaks the symmetry of the code compared to 
```rust
let first = slice.get(0).unwrap();
let second = slice.get(1).unwrap();
```

and makes the code not as obvious

I think we should just disable the lint for this line

Reply:
I agree 👍